charts/backstage/README.md.gotmpl

```diff
@@ -186,3 +186,31 @@ upstream:
         cors:
           origin: 'https://{{"{{"}}- include "janus-idp.hostname" . {{"}}"}}'
 ```
+
+
+### Vanilla Kubernetes compatibility mode
+
+In order to deploy this chart on vanilla Kubernetes or any other non-OCP platform, please make sure to apply following changes. Further customization may be required, depending on your exact Kubernetes setup:
+
+```yaml
+# values.yaml
+global:
+  host: # Specify your own Ingress host
+route:
+  enabled: false  # OpenShift Routes do not exist on vanilla Kubernetes
+upstream:
+  ingress:
+    enabled: true  # Use Kubernetes Ingress instead of OpenShift Route
+  backstage:
+    podSecurityContext:  # Vanilla Kubernetes doesn't feature OpenShift default SCCs with dynamic UIDs, adjust accordingly to the deployed image
+      runAsUser: 1001
+      runAsGroup: 1001
+      fsGroup: 1001
+  postgresql:
```

Comment:
This is the setting that I've just verified that works on IKS.
it forces the PostgreSQL run as `postgres` user and group (uid and guide 26) and it enables the `volumePermissions` initContinainer that fixes the permissions on the files in the data volume 

```yaml
primary:
  podSecurityContext:
    enabled: true
    fsGroup: 26
  containerSecurityContext:
    enabled: true
    runAsUser: 26
volumePermissions:
  enabled: true
  image:
    registry: registry.redhat.io
    repository: rhel9/postgresql-15
    tag: latest
```

Reply:
updated